maakay-bot.py

```diff
@@ -480,7 +481,7 @@ async def challenge_all(ctx):
                 role = "Contender"
             else:
                 role = "Referee"
-            embed.add_field(name="Challenge ID", value=challenge.uuid_hex)
+            embed.add_field(name=f"{challenge.title}", value=f">Challenge ID: {challenge.uuid_hex}\n>Amount: {convert_to_decimal(challenge.amount)}\n>Role: {role}")
```

Comment:
Looks like we need to add a space between > and text to get the desired response.
`.add_field(name=f"{challenge.title}", value=f"> ID: {challenge.uuid_hex}\n> Amount: {convert_to_decimal(challenge.amount)}\n> Role: {role}")`